Ryujinx.Ui.Common/App/ApplicationLibrary.cs

```diff
@@ -112,9 +112,9 @@ public void LoadApplications(List<string> appDirs, Language desiredTitleLanguage
                             {
                                 return;
                             }
-                        
+
```

Comment:
```suggestion

                            var fileInfo = new FileInfo(app);
```
If we create an instance of `FileInfo` here, we are able to simplify the steps below and check for symlinks as well.